### ISSUE TITLE
CommunityEntity.UI.Countdown.cs

### ISSUE THREAD

**Issue by OriginalRazor** [Facepunch/Rust.Community#59]
Can We get time spawn added in to display out H:M: S in CommunityEntity.UI.Countdown.cs

**Comment by TurEduard** [comment]
I agree, this is an important issue. Ideally.
I would also like to have access to current variable "step". Like %TIME_LEFT% only %CURRENT_STEP%

**Comment by TurEduard** [comment]
There are already developments in this direction since last year. But they were not accepted for release (((
https://github.com/Facepunch/Rust.Community/pull/56

**Comment by OriginalRazor** [comment]
Ua i see some valid good changes but problem is its not looking like a CommunityEntity probably more of a FriendsEntity

**Comment by TurEduard** [comment]
We can only wait and hope that the developers will accept this review. While this topic does not have a large scope, it is unlikely that this issue will be addressed(

**Comment by OriginalRazor** [comment]
Ya took them a month to fix the ping issue so only can imagine how much they care about community input :)

**Comment by TurEduard** [comment]
I personally waited more than 6 years for a scrollbar to be introduced into Cui. This only happened in this May update))))

**Comment by Kulltero** [comment]
i agree that it would be a huge addition (i'm definitely not biased), and i hope the PR to "supercharge" the current coundown component will be added soon, though there are other PRs currently further along i hope that the fact that this one is a bit simpler will help it get introduced faster 🙏 

i'd definitely reccommend to continue advocating for features you'd find useful, since they've recently added the scrolling feature its clear they are still looking onto here, even if they cant always prioritize it
